test/combineCoverage.js

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
We tend to put scripts that are only used in the pipelines under `build/azure-pipelines`

Reply:
I put this here because it's generally useful for folks to use as well locally to generate an all-up view of code coverage. If other people agree that this belongs under pipelines I'll gladly move it, just figured it's a little more discoverable here. Going to merge in now - but if anyone else sees this and agrees just let me know.